src/style.css

```diff
@@ -0,0 +1,26 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import url("./card.css");
+@import url("./stars.css");
+
+/* #region null */
+
+html,
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  font-family: "Roboto", sans-serif;
+
+  --border-color: #f3f3f3;
+  --button-color: #00acdc;
+  --main-text-color: #060b35;
+  --subtext-color: #616070;
+  --border-radius: 5px;
```

Comment:
You do not need all rules both to html and body selectors
html has to include only font-family and variables